cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,66 @@
 /// <reference types='cypress' />
 
-describe('Student Registration page', () => {
-  before(() => {
+// cypress/support/commands.js
+Cypress.Commands.add('generateUser', () => {
```

Comment:
Use faker, it'll give you realistic values for each variable. Randomise name and surname too.